### PR TITLE
Add basic EWS (Early Warning System) documentation.

### PR DESCRIPTION
**Pull request by mikenairn**
Doc can be viewed here https://github.com/integr8ly/delorean/blob/c1c4ba49cd523527159ba7b13b91fe233c7f74d7/docs/ews/README.md

Jira:

* https://issues.redhat.com/browse/DEL-249

### CODE REVIEW

File: docs/ews/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+All the usual CI (prow) jobs will execute against the EWS PRs, but some jobs have been updated to prevent accidental inclusion of files/updates solely for the testing of product pre release versions:
+
+* ci/prow/manifest - fails if image mirror mapping files exists or any references to internal registries exist in any CSVs. It would be expected that this will fail for all pre release PRs.
+
+
+### Adding new products
+
+To add a new component/product to the EWS pipelines, it must:
+
+* Produstized (Go through the RH internal build system)
```
Comment (STEPHINRACHEL) [inline docs/ews/README.md:95]: Typo in Productized

**Comment by STEPHINRACHEL** [comment]
Great work!
Happy to have others review as well
/lgtm

File: docs/ews/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+
+### Release Discovery Pipeline
+
+The main component of the EWS is the release discovery pipeline, the currently configured pipelines can all be found [here](https://master-jenkins-csb-intly.cloud.paas.psi.redhat.com/job/Delorean/job/ews/).
+Every component/product that is enabled on the EWS has it's own discovery job, and it's role is to search for new versions of a particular product, and update the installer([integreatly operator](https://github.com/integr8ly/integreatly-operator)).
+
+It has the following responsibilities:
+* Discovery of new versions.
+* Create a new branch against the installer if one does not already exist (\<product-name\>-next-\<version\> i.e. 3scale-next-0.5.2)
```
Comment (wei-lee) [inline docs/ews/README.md:23]: what about information around if the product is GA'ed? Is that a label on the PR, or in the description of the PR?

Comment (mikenairn) [reply docs/ews/README.md:23]: Further down the doc, we are using labels on the PR.

File: docs/ews/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+The triggered CI is already setup to allow the testing of pre release versions of products by applying the delorean pull secret as described [here](#installation) 
+
+#### PR labels
+
+As with all prow enabled repos, labels are used extensively to control what happens, and give the status of the state of these PRs.
+
+Along with the usual prow workflow labels there also these labels used by delorean:
+
+* do-not-merge/product-pre-release - Used as a tide merge blocker, applied to all EWS PRs that include products that are not yet GA
```
Comment (wei-lee) [inline docs/ews/README.md:81]: when/how this label will be removed?

Comment (mikenairn) [reply docs/ews/README.md:81]: These 2 labels are toggled by the discovery job, it manages all of this.

Comment (mikenairn) [reply docs/ews/README.md]: Add additional note to show that automation manages these labels

File: docs/ews/README.md (hunk 2 above)
Comment (mikenairn) [reply docs/ews/README.md:23]: Added extra line here to show that it manages labels relating to GA status

**Comment by laurafitzgerald** [comment]
/lgtm
/approve
This is a great doc!

**Pull request merged**